src/langsmith/self-host-blob-storage.mdx

```diff
@@ -44,7 +44,12 @@ To authenticate to [Amazon S3](https://aws.amazon.com/s3/), you will need to cre
   "Statement": [
     {
       "Effect": "Allow",
-      "Action": "s3:*",
+      "Action": [
```

Comment:
In the description before this JSON, we say "you will need to create an IAM policy granting admin permissions on your bucket"... should we remove `admin`?

... you will need to create an IAM policy granting the following permissions on your bucket."